cdap-docs/developer-manual/source/index.rst

```diff
@@ -50,6 +50,10 @@ CDAP Developer Manual
 - |pipelines|_ A capability of CDAP that combines a user interface with back-end services
   to enable the **building, deploying, and managing of data pipelines.**
 
+.. |cloud-runtimes| replace:: **Cloud Runtimes:**
+.. _cloud-runtimes: cloud-runtimes/index.html
+
+- |cloud-runtimes|_ Configuring pipelines to run in the cloud instead of on premise.
```

Comment:
Are we not planning to document how to run non-"Data Pipeline" workflows in the cloud?

Reply:
no. Though you can technically do this for workflows, the target use case for 5.0.0 is batch pipelines. It will be mentioned somewhere, but not emphasized